testsuite/gnat2goto/tests/range_second/test.opt

```diff
@@ -0,0 +1,2 @@
+ALL XFAIL Doesn't crash anymore but cbmc does not perform range assertion checks 
```

Comment:
I don't think there's any relevant test for this. The range of this PR was for a very particular piece of code (using enum symbols as range constraints) to not crash. This has been fixed, but gnat2goto suffers from a different problem now (it doesn't handle constraints of subtypes correctly). I can't easily think of something that can exercise the fix without falling on its face because of the subtype issue.